config/jobs/gardener-extension-registry-cache/gardener-extension-registry-cache-e2e-kind.yaml

```
@@ -0,0 +1,34 @@
+presubmits:
+  gardener/gardener-extension-registry-cache:
+  - name: pull-gardener-extension-registry-cache-e2e-kind
+    cluster: gardener-prow-build
+    always_run: true
+    decorate: true
+    decoration_config:
+      timeout: 60m
+      grace_period: 15m
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    annotations:
+      description: Runs end-to-end tests for gardener-extension-registry-cache developments in pull requests
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:v20230810-706aedbe3b-master
```

```suggestion
      - image: eu.gcr.io/gardener-project/ci-infra/krte:v20230818-878cbf0-1.21
```
Registry cache extension is built with Go 1.21 now, so we will need a `krte` image with Go 1.21 too.